tests/acceptance/features/bootstrap/WebDav.php

```diff
@@ -5373,9 +5373,9 @@ public function findEntryFromPropfindResponse(
 				$entryPath = $multistatusResult['value'][0]['value'];
 				if (OcisHelper::isTestingOnOcis() && $method === "REPORT") {
 					if ($entryNameToSearch !== null && str_ends_with($entryPath, $entryNameToSearch)) {
-						return $multistatusResults;
+						return $multistatusResult;
 					} else {
-						$spaceId = (WebDavHelper::$SPACE_ID_FROM_OCIS) ? WebDavHelper::$SPACE_ID_FROM_OCIS : WebDavHelper::getPersonalSpaceIdForUser(
+						$spaceId = (WebDavHelper::$SPACE_ID_FROM_OCIS) ?: WebDavHelper::getPersonalSpaceIdForUser(
```

Comment:
Why removing here?
This ternary operator has no value to return when the condition is true (will return `1` by default).
Don't we want to return `space_id_from_ocis` if the condition is true?
```suggestion
						$spaceId = (WebDavHelper::$SPACE_ID_FROM_OCIS) ? WebDavHelper::$SPACE_ID_FROM_OCIS  : WebDavHelper::getPersonalSpaceIdForUser(
```